source/simulator/stokes_matrix_free.cc

```diff
@@ -1837,7 +1837,18 @@ namespace aspect
                                                           velocity.get_symmetric_gradient (q);
             VectorizedArray<double> pres = pressure.get_value(q);
             VectorizedArray<double> div = trace(sym_grad_u);
```

Comment:
These two variables could be `const` as well, I believe.

As a general rule, mark everything as `const` that can be marked as `const`.